sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -169,7 +175,7 @@ Beyond the scenarios discussed, the Azure Event Hubs client library offers suppo
 - [Consume events from an Event Hub partition](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample7_ConsumeEvents.cs)  
   An introduction to consuming events, using a simple Event Hub consumer.
   
-- [Consume events from an Event Hub partition in batches](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample8_ConsumeEventsByBatch)  
+- [Consume events from an Event Hub partition in batches](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample8_ConsumeEventsByBatch)   
   An example of consuming events, using a batch approach to control throughput.
   
 - [Consume events from a known position in the Event Hub partition](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample9_ConsumeEventsFromAKnownPosition)  
```

Comment:
Also about contributing section - apparently the way it is currently written it is standardized by CELA and has to look exactly as it does in the sample one here  https://github.com/Azure/azure-sdk/blob/master/docs/README-EXAMPLE.md

Reply:
Reformatted to include the text.